### PR TITLE
add target for new temp-humidity module

### PR DESCRIPTION
**Pull request by jamesadevine**

### CODE REVIEW

**Comment by jamesadevine** [comment]
will need to update jacdac-stm32x0 submodule

File: targets/jm-temp-humidity-18-1.0/config.mk [hunk 1]
```diff
@@ -0,0 +1,2 @@
+MCU = STM32F030x4
```
Comment (mmoskal) [inline targets/jm-temp-humidity-18-1.0/config.mk:1]: wrong mcu?

Comment (jamesadevine) [reply targets/jm-temp-humidity-18-1.0/config.mk]: Sorry, didn't commit everything. Should be correct now

**Review by mmoskal — COMMENTED** [review]
could you add jm-temp-humidity-18-1.0A to DROP_TARGETS? so it gets built in gh action

**Comment by jamesadevine** [comment]
> could you add jm-temp-humidity-18-1.0A to DROP_TARGETS? so it gets built in gh action

Done!

**Comment by mmoskal** [comment]
you might have missed the .mk file?
https://github.com/microsoft/jacdac-msr-modules/pull/1/checks?check_run_id=1982284130#step:8:811

**Comment by jamesadevine** [comment]
It's in the PR to the jacdac-stm33x0 repository:

https://github.com/microsoft/jacdac-stm32x0/pull/15/files

**Comment by mmoskal** [comment]
upper case/lower case "stm"

**Comment by jamesadevine** [comment]
Flip! Thanks, will push.

I'm in two minds as to whether all file systems should be case insensitive :wink:

**Pull request merged**